.github/workflows/build.yml

```diff
@@ -865,6 +865,37 @@ jobs:
           makeLatest: true
           tag: ${{ steps.version.outputs.version }}
           token: ${{ secrets.NETDATABOT_GITHUB_TOKEN }}
+      - name: Checkout netdata main Repo # Checkout back to netdata/netdata repo to the update latest packaged versions
+        id: checkout-netdata
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.NETDATABOT_GITHUB_TOKEN }}
+      - name: Init python environment for publish release metadata
+        uses: actions/setup-python@v4
+        id: init-python
+        with:
+          python-version: "3.9"
```

Comment:
Can we bump this to a more recent version of Python? It’s unfortunately not something that Dependabot can reliably track, and AFAICT your scripts will all work on Python 3.12, so having it as recent as possible gives as much time as possible before we run into potential issues because of an old version.